docs/tools/third-party/tavily.md

```diff
@@ -0,0 +1,118 @@
+# Tavily
+
+The [Tavily MCP Server](https://github.com/tavily-ai/tavily-mcp) connects your
+ADK agent to Tavily's AI-focused search, extraction, and crawling platform. This
+gives your agent the ability to perform real-time web searches, intelligently
+extract specific data from web pages, and crawl or create structured maps of
+websites.
+
+## Use cases
+
+- **Real-Time Web Search**: Perform optimized, real-time web searches to get
+  up-to-date information for your agent's tasks.
+
+- **Intelligent Data Extraction**: Extract specific, clean data and content from
+  any web page without needing to parse the full HTML.
+
+- **Website Exploration**: Automatically crawl websites to explore content or
+  create a structured map of a site's layout and pages.
+
+## Prerequisites
+
+- [Sign up for a Tavily account](https://app.tavily.com/) to obtain an API key.
```

Comment:
Make links text as short as possible, and avoid starting a sentence with a link:

Sign up for a [Tavily account](https://app.tavily.com/)